cli/src/main/java/com/liferay/blade/cli/util/BladeUtil.java

```diff
@@ -318,32 +319,80 @@ public static String getManifestProperty(Path pathToJar, String propertyName) th
 		}
 	}
 
-	public static Map<String, Object> getProductInfos() {
-		return getProductInfos(false, null);
+	@SuppressWarnings("unchecked")
+	public static ProductInfo getProductInfo(String productKey) {
```

Comment:
ProductInfo should be renamed to ReleaseInfo, since the information is coming from releases.json